src/Core/src/Handlers/Button/ButtonHandler.Windows.cs

```diff
@@ -9,6 +9,7 @@ public partial class ButtonHandler : ViewHandler<IButton, Button>
 	{
 		PointerEventHandler? _pointerPressedHandler;
 		PointerEventHandler? _pointerReleasedHandler;
+		bool _isPressed;
```

Comment:
Do we need to apply this to ImageButton as well?

Reply:
Yeah, good idea